oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,15 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+
+  $(".dropdown").hover(
```

Comment:
Need your confirmation over here, you added this for that dropdown button of session links on the navbar?